### ISSUE TITLE
Nginx 413 error message

### ISSUE THREAD

**Issue by jace** [hasgeek/hasdev#6]
When a user tries to upload a very large image on Hasjob, nginx errors out with HTTP 413 even before Hasjob is invoked. This means we can't display a meaningful error message from Hasjob and need to handle this in Nginx itself.

This is similar to the serverdown message we show users when uwsgi goes away (HTTP 502).

**Comment by miteshashar** [comment]
If the operation is eventually going to fail, why not do error handling on client side with js, instead of doing it after the whole big file upload?

**Comment by jace** [comment]
Good idea. I haven't investigated client side validation for image uploads.

FWIW, Hasjob has a long-pending switch to Imgee, but Imgee needs cleanup before it can be put in the spotlight with Hasjob.

**Comment by miteshashar** [comment]
Yeah, indeed. I see the connection.
Though, till imgee is cleaned up, the file uploading implementation in WTForms in baseframe(IIRC baseframe would have that implementation) could potentially be the right place to factor this in.
